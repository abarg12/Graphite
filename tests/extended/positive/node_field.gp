node<int> n;
n.flag = true;
#bool f = n.flag;
#printf(f);
#n.data = 5;
#int int_value = n.data;
#printf(int_value);

void node_in_func() {
    node<string> m;
    m.flag = false;
    bool f = m.flag;
    printf(f);
    #int g = n.data;
    #printf(g);
    return;
}

node_in_func();