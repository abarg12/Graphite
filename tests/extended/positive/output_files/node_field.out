1
5
5
7
0
7
I really hope this works
if only grammatical symbols were allowed in strings in our language lol this has no personality
1
